lectures/dynamic_programming/mccall_model_with_separation.md

```diff
@@ -284,28 +285,29 @@ We'll use the default parameterizations found in the code above.
 
 ```{code-cell} julia
 # a default utility function
-u(c, σ) = (c^(1 - σ) - 1) / (1 - σ)
+u(c, sigma) = (c^(1 - sigma) - 1) / (1 - sigma)
 
 # model constructor
-McCallModel = @with_kw (α = 0.2,
-    β = 0.98, # discount rate
-    γ = 0.7,
-    c = 6.0, # unemployment compensation
-    σ = 2.0,
-    u = u, # utility function
-    w = range(10, 20, length = 60), # wage values
-    dist = BetaBinomial(59, 600, 400)) # distribution over wage values
+function McCallModel(; alpha = 0.2,
+                     beta = 0.98, # discount rate
+                     gamma = 0.7,
+                     c = 6.0, # unemployment compensation
+                     sigma = 2.0,
+                     u = u, # utility function
```

Comment:
My preference here is to replace this `u = u` with `u = (c, sigma) ->  (c^(1 - sigma) - 1) / (1 - sigma)` from above, then get rid of that line above as well on the utility funciton
If it is necessary to directly use the utility function lower in the code, you can do it by going `mcm.u(...)` etc. as a function on that type.